### PR TITLE
solve format issues 

### LINKED ISSUE #46: Additional Tests
- have code coverage communicated with codecov.io button in README
- Add additional test cases, need to achieve 90% branch coverage or higher

### PR DESCRIPTION
**Pull request by clsu22**
partially fixs #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
+```python 
+from pyimager import pyimager
+
+pyimager.reducolor(0, 'tests/mandrill.jpg', 'tests/mandrill_new.jpg')
+```
+
+```python
+from pyimager import pyimager
+
+pyimager.imgfilter("images/bear.jpg", "blur", 0.4)
```
Comment (Keanna-K) [inline README.md]: Changing my example to a photo that is in our repo. 
```suggestion
pyimager.imgfilter("images/mandrill.jpg", "blur", 0.4)
```

File: README.md [hunk 2]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
-## Usage:
+## Usage Examples 
 
-``` python
+```python
 from pyimager import pyimager
+
+pyimager.circropper("images/bear.jpg", 0)
+```
+![](images/bear.jpg)
```
Comment (Keanna-K) [inline README.md]: changing to a photo that is present in our repo. 

```suggestion
pyimager.circropper("images/mandrill.jpg", 0)
```

Comment (Keanna-K) [inline README.md]:
```suggestion
pyimager.circropper("images/bear.jpg", 0)
```
 
```

File: README.md [hunk 3]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
+
+pyimager.circropper("images/bear.jpg", 0)
+```
+![](images/bear.jpg) 
+![](images/result2.png)
+
+```python 
+from pyimager import pyimager
+
+pyimager.reducolor(0, 'tests/mandrill.jpg', 'tests/mandrill_new.jpg')
```
Comment (Keanna-K) [inline README.md]: we should place all our images in the images folder. 
```suggestion
pyimager.reducolor(0, 'images/mandrill.jpg', 'images/mandrill_new.jpg')
```

File: README.md [hunk 4]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
+```python
+from pyimager import pyimager
+
+pyimager.imgfilter("images/bear.jpg", "blur", 0.4)
 ```
 
+```python
+from pyimager import pyimager
+
+pyimager.reducolor(0, 'tests/mandrill.jpg', 'tests/mandrill_new.jpg')
```
Comment (Keanna-K) [inline README.md]:
```suggestion
pyimager.reducolor(0, 'images/mandrill.jpg', 'images/mandrill_new.jpg')
```

File: README.md [hunk 5]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
+    - matplotlib = "^3.2.0"
 
-## Usage:
+## Usage Examples 
 
-``` python
+```python
 from pyimager import pyimager
+
+pyimager.circropper("images/bear.jpg", 0)
```
Comment (Keanna-K) [inline README.md]:
```suggestion
pyimager.circropper("images/mandrill.jpg", 0)
```

File: README.md (hunk 2 above)
Comment (Keanna-K) [inline README.md]:
```suggestion
![](images/mandrill.jpg) 
```

Comment (clsu22) [reply README.md]: I have added two images in README, both bear, and you just give me one suggestion on changing the bear to mandrill, which will cause an unmatched result in this example. Would you mind changing this suggestion to "delete this image", and add one more suggestion on "delete the image after this one, 'images/result2.jpg'"? Thank you.

File: README.md [hunk 6]
```diff
@@ -44,18 +44,39 @@ pip install -i https://test.pypi.org/simple/ pyimager
...
+```
+![](images/bear.jpg) 
+![](images/result2.png)
+
```
Comment (Keanna-K) [inline README.md]:
```suggestion
![](images/result2.png)
os.remove('images/results2.jpg')
```

**Pull request merged**